README.md

```diff
@@ -20,6 +20,28 @@ This is an addon that allows to export Vaadin's grid data to some formats like E
 
 [Available in Vaadin Directory](https://vaadin.com/directory/component/grid-exporter-add-on)
 
+### Maven install
+
+Add the following dependencies in your pom.xml file:
+
+```xml
+<dependency>
+	<groupId>org.vaadin.addons.flowingcode</groupId>
```

Comment:
Code block is monospaced, please use consistent indentation.
![image](https://user-images.githubusercontent.com/11554739/191255392-3eb66da1-b094-4d6a-9e09-b780ee49ca16.png)